src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractExtensionMethodImportCompletionProvider.cs

```diff
@@ -67,7 +67,12 @@ protected override void LogCommit()
                     var items = await getItemsTask.ConfigureAwait(false);
 
                     var receiverTypeKey = SymbolKey.CreateString(receiverTypeSymbol, cancellationToken);
-                    completionContext.AddItems(items.Select(i => Convert(i, receiverTypeKey)));
+                    var completionItems = items.Select(item => Convert(item, receiverTypeKey));
+                    var isParenthesisCompletionEnable = IsAutoAddParenthesisBySemicolonEnabled(completionContext.Document);
```

Comment:
Although this is OK functionally, and shouldn't impact perf due to much smaller amount of items (compared to unimported types), we should still avoid doing this for all items, and delay the check until a item is being committed.